Note that Firefox works too. (#705)
github/docs: content/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests.md

{% tip %}

> **Tip:** If you're using Chrome or Firefox, you can also copy-and-paste images directly into the box.

Author: ```suggestion
**Tip:** If you're using Chrome, Firefox, or any Chromium-based browser (like the new Edge), you can also copy-and-paste images directly into the box.
```

> **Tip:** If you're using Chrome, Firefox, or any Chromium-based browser (like the new Edge), you can also copy-and-paste images directly into the box.

Reviewer: ```suggestion
**Tip:** In many browsers, you can also copy-and-paste images directly into the box.
```

Thanks for opening a PR @KTibow! Since the list has grown and we want to keep things evergreen 🌲, let's keep it vague and generalized. That way, we don't have to worry about constantly updating the list. Let me know when you've accepted the suggestion and I'll get this merged in!

Author: Added that.
